src/components/Avatar/Avatar.style.ts

```diff
@@ -51,10 +45,13 @@ export const avatarStyle = ({
   width: ${sizeBasedOnProp(size)};
   height: ${sizeBasedOnProp(size)};
   border-radius: 100%;
+  border: ${rem(1)} solid ${theme.utils.getColor('lightGrey', 100)};
+  box-sizing: border-box;
```

Comment:
Not relevant to this, but I think we are not consistent in applying this to all of our components. Shouldn't we be?